griptape/utils/prompt_stack.py

```diff
@@ -16,7 +16,7 @@ class PromptStack:
 
     @dataclass
     class Input:
-        content: str
+        content: str | list[dict]
```

Comment:
Setting up for other [content types](https://platform.openai.com/docs/guides/vision/quick-start)